sdk/android/analytics.md

```diff
@@ -658,8 +658,9 @@ public void addCustomEndpointAttribute() {
 #### Assigning User IDs to Endpoints
 Assign user IDs to endpoints by doing either of the following:
 
-Manage user sign-up and sign-in with Amazon Cognito user pools.
-Use the Amazon Pinpoint client to assign user IDs without using Amazon Cognito user pools.
+* Manage user sign-up and sign-in with Amazon Cognito user pools.
+* Use the Amazon Pinpoint client to assign user IDs without using Amazon Cognito user pools.
+
 Amazon Cognito user pools are user directories that make it easier to add sign-up and sign-in to your app. When the AWS Mobile SDKs for iOS and Android register an endpoint with Amazon Pinpoint, Amazon Cognito automatically assigns a user ID from the user pool. For more information, see Using Amazon Pinpoint Analytics with Amazon Cognito User Pools in the Amazon Cognito Developer Guide.
```

Comment:
Should "see" be folllowed by a link or links to [Using Amazon Pinpoint Analytics with Amazon Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-pinpoint-integration.html) and the [Amazon Cognito Developer Guide](https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html)?

Reply:
Reading that link, I don't actually see the information noted in the graf or above bullets. Where are readers to go to learn more?